grakn-factory/janus-factory/src/test/java/ai/grakn/factory/TxFactoryJanusTest.java

```diff
@@ -191,6 +193,29 @@ public void testGraphNotClosed() throws InvalidKBException {
         graph.putEntityType("A Thing");
     }
 
+    @Test
+    public void whenSeveralFactoriesCreateTheSameKeyspace_NoErrorOccurs() throws Exception {
```

Comment:
I suspect this case is already covered by `FactoryBuilderTest#whenBuildingFactoriesPointingToTheSameKeyspace_EnsureSingleFactoryIsReturned` and `GraknTest#whenGettingSessionForSameKeyspaceFromMultipleThreads_EnsureSingleSessionIsReturned`

What prompted the need to create this test?